shared/common-adapters/avatar.desktop.js

```diff
@@ -19,9 +19,20 @@ export default class Avatar extends Component {
     this.state = {avatarLoaded: false}
   }
 
+  _createUrl (): ?string {
+    if (this.props.url) {
+      return this.props.url
+    } else if (this.props.username) {
```

Comment:
It's nicer to have that logic in one place, otherwise we have multiple spots that know how to build a avatar url from a username.

It might make sense to move this logic, but it's definitely easier and nicer to have the avatar know how to get an avatar image given a username